[DOCS-4039] Add CSI doc (#15590)
DataDog/documentation: content/en/security_platform/cloud_siem/cloud_security_investigator.md

---
title: Cloud Security Investigator
kind: documentation
further_reading:
- link: "/cloud_siem/explorer/"
  tag: "Documentation"
  text: "Learn about the Security Signals Explorer"
---

{{< beta-callout url="#" btn_hidden="true">}}

>   The Cloud Security Investigator is currently in beta.

Reviewer: ```suggestion
  The Cloud Security Investigator is in beta.
```

{{< /beta-callout >}} 

<div class="alert alert-warning">Cloud Security Investigator currently only supports AWS Cloudtrail logs.</div>

## Overview

When a security signal alerts on suspicious activity by a user, common questions asked as part of the investigation could include:

- Is the user accessing other accounts? 
- What other actions did the user take around that specific time frame? 
- What are all the actions taken on a resource by the user?

> For example, you received a security signal that someone changed the configuration of an AWS S3 bucket so that it is accessible by everyone, but the action was taken by an assumed role. To investigate, you might want to look into who took the action and what other activities they have taken recently, as that could indicate compromised credentials.

Reviewer: ```suggestion
For example, suppose you receive a security signal that someone changed the configuration of an AWS S3 bucket so that it is accessible by everyone, but the action was taken by an assumed role. To investigate, look into who took the action and what other activities they did recently, as that could indicate compromised credentials.
```
